src/Env.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button";
+import { FormGroup } from "@patternfly/react-core/dist/esm/components/Form";
+import { FormHelper } from "cockpit-components-form-helper.jsx";
+import { Grid } from "@patternfly/react-core/dist/esm/layouts/Grid";
+import { TextInput } from "@patternfly/react-core/dist/esm/components/TextInput";
+import { MinusIcon } from '@patternfly/react-icons';
+import cockpit from 'cockpit';
+
+import * as utils from './util.js';
+
+const _ = cockpit.gettext;
+
+export function validateEnvVar(env, key, validationFailed) {
+    const delta = { ...validationFailed };
+
+    switch (key) {
+    case "envKey":
+        if (utils.isEmpty(env.envKey))
+            delta.envKey = _("Required");
+        break;
+    case "envValue":
+        if (utils.isEmpty(env.envValue))
+            delta.envValue = _("Required");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-110243-108e1b8a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)